networkx/conftest.py

```diff
@@ -209,13 +209,14 @@ def add_nx(doctest_namespace):
 except ImportError:
     has_sympy = False
 
-
 # List of files that pytest should ignore
 
 collect_ignore = []
 
 needs_numpy = [
     "algorithms/approximation/traveling_salesman.py",
+    "algorithms/bipartite/matching.py",
+    "algorithms/bipartite/tests/test_matching.py",
```

Comment:
These lines in networkx/conftest.py also have to be updated to match the new module name for the envy-free-matching stuff. Since the now smaller `matching.py` doesn't use `numpy` (I think), you should remove that name and add the envy-free module name.  Similarly for `needs_scipy` below.